Blish HUD/GameServices/DebugService.cs

```diff
@@ -49,10 +49,6 @@ public class DebugService : GameService {
                 FileNameKind      = FilePathKind.Absolute,
                 ArchiveFileKind   = FilePathKind.Absolute,
                 FileName          = Path.Combine(logPath, "blishhud.${cached:${date:format=yyyyMMdd-HHmmss}}.log"),
-                ArchiveFileName   = Path.Combine(logPath, "blishhud.${cached:${date:format=yyyyMMdd-HHmmss}}.{#}.log"),
-                ArchiveDateFormat = "yyyyMMdd-HHmmss",
-                ArchiveAboveSize  = MAX_LOG_SIZE,
```

Comment:
You probably want to keep `ArchiveAboveSize = MAX_LOG_SIZE,`

Reply:
Hmmm ... but then we should use not only `Date` but `DateAndSequence` in the filename?
Aka what would happen if we surpass the file size?

Reply:
Would NLog simply create a new file with the current time? That way it would be very unlikely that we surpass 1MB in less than a second.

Reply:
NLog will by default archive using sequence-no:

```
blishhud.20010101-010101.log <- Active
blishhud.20010101-010101.0.log
blishhud.20010101-010101.1.log
blishhud.20010101-010101.2.log
```

Reply:
Nice!
Thank you very much for your help! Much appreciated!